src/screens/ExchangeModal.js

```diff
@@ -304,10 +304,14 @@ export default function ExchangeModal({
           return;
         }
 
-        const callback = () => {
+        const callback = (success = true, errorMessage = null) => {
```

Comment:
Personal preference = default values to be falsey because it reflects the behavior of undefined